### ISSUE TITLE
Open the app from the Android notification

### ISSUE THREAD

**Issue by zstadler** [IsraelHikingMap/Site#1613]
## What I expect to happen
When the GPS is active, and I touch the Israel Hiking Map notification on Android, I would like the app to move to the foreground.

## What really happened
The App info is opened instead

## What I did that caused the issue - step by step

1. Activate the GPS
2. Show the Android notification list
3. Touch the Israel Hiking Map notification 

## My environment
* Operating System and version: Android

**Comment by HarelM** [comment]
the following describes the issue:
https://www.titanwolf.org/Network/q/7ea03b83-5f35-416a-a622-3146c85d23b2/y
TL;DR: when not setting a small icon the notification will open the settings and not the app (god knows why).
This apparently was introduced with this commit: df14d43ecc7b5a73669ab8b2091afc07fcbb1e85
When debugging the android application it seems that the icon I added in this commit is returning 0, which is super weird as the icon is showing in the notification bar so I don't know how this is possible...
When not setting any icon the default icon is used by the plugin and it works as expected.
I need to understand how to copy the icon to the right place.
If I can't manage to do it, I prefer to have the icon show up and open setting over not showing an icon...

**Comment by HarelM** [comment]
Seems like when Cordova is copying the resources it copies the splash screen icon to `screen` folder inside `drawable` folder, which should facilitate for this... I hope...
